clients.md

```diff
@@ -26,35 +25,59 @@ Basing on best practices.
+But when it comes to reality, it usually goes not as perfect as planned. Actually, even worse than that: [CHAOS report](https://www.projectsmart.co.uk/white-papers/chaos-report.pdf) claims that the average success rate is only 16.2% for software projects, meaning those that were completed on-time and on-budget. And speaking of the challenged ones, 52.7% costed 189% of their original estimate.
+
+Apart from this numbers, a client gets only and solely what is put on paper. So it means that project plan does not imply any of the following:
 
-Apart from this numbers, client gets their limitations: the delivery will contain only what is put on paper and not a tiny bit more. So it means that the project does not imply any of the following:
 1. Additional requests from the client;
 2. Unforeseen issues being handled;
 3. Sudden bright ideas;
 4. Even sane requirements of reasonability.
 
-This four issues form the majority of frictions between a client and a provider during a project. Apparently you might have already been exposed to them.
+To recup: this casual approach does not motivate a contractor to deliver a tiny bit more to a project than fixed in an agreement. It also works the way that a contractor will always try to get as much money as they can in order to, among other things, compensate high risks that are essential part of this methodology. 
+
+When you have a casual project triangle restrictions, it means you never end up with full and clear understanding why this have costed that much, and how close you are to exceeding the budget or time.
+
+This forms the majority of frictions between a client and a provider during a project. Apparently you might have already been exposed to them.
+
+#### Hello, wemake a service
+
+In order to overcome the disadvantages of conventional waterfall model, we introduce our own management approach. Following our naming pattern, which you might have already noticed, we call it "wemake a service". It leaves only two options for project limitation:
+
+- project scope,
+- or fixed due time and budget.
+
+Although it might be something that does not look perfectly sensible from the first glance, practically it means that all parties care about the primary goal first and prioritize major things against less important ones to deliver a product that addresses the core issue.
+
+The rule of thumb is that you cannot have both limitation options from above been set the same time. 
+
+#### How it works
+
+1. We design and document the first module of delivery basing on your priorities.
+2. The module is broken down to small pieces. Each one has a declared price which comprises of two parts: what we pay to a developer, and our fee.
+3. We deliver the pieces one by one. Each delivery has clear documentation. And you always have an access to our task management tool to check what is happening.
```

Comment:
Not only documentation, but also tests and deployment tutorials.
And every client has an access to everything: source code, tasks, CI jobs, code statics, tests coverage, LOC, HOC, code quality reports.